src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -215,7 +218,7 @@ def arbeitsl_geld_2_eink_anr_frei_m(
     # condition.
     eink_erwerbstätigkeit = bruttolohn_m + eink_selbst_m
 
-    if anz_kinder_bis_17_hh > 0:
+    if anz_kinder_bis_17_bg > 0 or _eink_st_kinderfreib_anz_ansprüche > 0:
```

Comment:
§11b SGB 2 Abs. 3: [...] die entweder mit mindestens einem minderjährigen Kind in Bedarfsgemeinschaft leben oder die mindestens ein minderjähriges Kind haben [...]

Reply:
Put ref in the code?

Reply:
It is, just wanted to make the review easier for you because this comes out of the blue (we couldn't depict this before the recent changes).